lib/generators/install/templates/index.coffee.erb

```diff
@@ -0,0 +1,8 @@
+#= require cable
+#= require_self
+#= require_tree .
+
+@<%= application_name %> ?= {}

```

Comment:
Fair point. I don't think any frameworks recommend setting arbitrary property names on your global app object though so minimally we should namespace them (`AppName.subscriptions.appearance` instead of `AppName.appearance`).

Also, we can make storing your subscriptions on `Cable` the recommended way if we want. The API isn't final, and it might make your code more obvious to other readers. If I came across `Cable.subscriptions.appearance.doSomething()` in a non-channel file, I'd be pretty darn sure it was Action Cable related. `AppName.appearance.doSomething()`, not so sure.

Finally, if we're trying to lead newer developers in the right direction, we shouldn't make subscriptions global at all.

This example in the README
```coffee
App.appearance = App.cable.subscriptions.create "AppearanceChannel",
  appear: ->
    @perform 'appear', ...

$(document).on 'page:change', ->
  App.appearance.appear()
```
could be rewritten as:
```coffee
appearance = App.cable.subscriptions.create "AppearanceChannel",
  appear: ->
    @perform 'appear', ...

$(document).on 'page:change', ->
  appearance.appear()
```

And app makers can decide to expose them as globals if/when they need access from outside.